src/Person.tsx

```diff
@@ -142,6 +161,40 @@ export default function Person(props: Props) {
     const value: string = e.target.value;
     covidEventsState[name].set(value);
   };
+  const handleSubmit = () => {
+    datesMissing.set(
```

Comment:
You can just do datesMissing[key].set(...)

Reply:
Ah yep, redone.